src/main/java/com/snowflake/kafka/connector/internal/telemetry/SnowflakeTelemetryService.java

```diff
@@ -151,15 +161,17 @@ public void reportKafkaPartitionStart(final SnowflakeTelemetryBasicInfo pipeCrea
    * {
    *  "app_name": "<connector_app_name>",
    *  "task_id": 1,
+   *  "snowflake.ingestion.method": "<Enum Ordinal>" for {@link IngestionMethodConfig}
    * }
    * </pre>
    *
    * @return An ObjectNode which is by default always created with certain defined properties in it.
    */
-  protected ObjectNode getObjectNode() {
+  protected ObjectNode getDefaultObjectNode(IngestionMethodConfig ingestionMethodConfig) {
     ObjectNode msg = MAPPER.createObjectNode();
     msg.put(APP_NAME, getAppName());
     msg.put(TASK_ID, getTaskID());
+    msg.put(SnowflakeSinkConnectorConfig.INGESTION_METHOD_OPT, ingestionMethodConfig.ordinal());
```

Comment:
Is this possible we put the string instead of ordinal? Ordinal is not straight forward and we have to remember the mappings

Reply:
I had kept string originally but I guess we only have two ingestion types(right now) so its not terrible to remember this. I was thinking we could save some storage since these are constants and gets added to all telemetry.